File: PythonToJava/sort_input.py

def main():

    n = int(input('Number of integers to sort: '))
    numbers = []
    for i in range(n):
        number = int(input(f'Enter item {i+1}: '))
        numbers.append(number)
    
    numbers.sort()
    
    for i in range(n):
        print(numbers[i])

if __name__ == "__main__":
    main()

File: PythonToJava/SortInput.java
import java.util.Arrays;
import java.util.Scanner;

public class SortInput {
    
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        
        int n = sc.nextInt();
        int[] numbers = new int[n];
        for (int i = 0; i < numbers.length; i++) {
            System.out.printf("Enter item %d: ", i+1);
            int number = sc.nextInt();
            numbers[i] = number;
        }
        sc.close();

        Arrays.sort(numbers);

        for (int i = 0; i < numbers.length; i++) {
            System.out.println(numbers[i]);
        }
    }
}
